blocks-touch/b-slider/b-slider.js

```diff
@@ -330,8 +333,15 @@
 
         _onPointerDown: function(e) {
 
+            var slider = this;
+
             e = e.originalEvent;
 
+            // каскадная подписка на move / up
+            slider._page
+                .on( 'pointermove', slider._bonPointerMove )
```

Comment:
It's better to use `this.bindToWin` here. This will guarantee that no event-listeners would stay in memory, if block would be destructed before `_pointerUp` happened.